packages/katello/katello/hostname-change.rb

```diff
@@ -400,8 +400,8 @@ def run
         if success
           STDOUT.puts result
           STDOUT.puts 'Restarting puppet services'
-          run_cmd('/sbin/service puppet restart')
-          run_cmd('katello-service restart --only puppetserver')
+          run_cmd('/bin/systemctl puppet restart')
```

Comment:
The order is reversed. We can also make it safer by using `try-restart` so it doesn't require Puppet to be running.
```suggestion
          run_cmd('/bin/systemctl try-restart puppet')
```